coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java

```diff
@@ -544,6 +545,19 @@ public boolean areActiveQuestionnaires(List<AnswerHeader> answerHeaders) {
         return false;
     }
 
+    public boolean areAnsweredQuestionnaires(List<AnswerHeader> answerHeaders) {
+        for (AnswerHeader answerHeader : answerHeaders) {
+            if (answerHeader.isActive()) {
+                for (Answer answer : answerHeader.getAnswers()) {
+                    if (StringUtils.isNotEmpty(answer.getAnswer())) {
```

Comment:
Ohhhh...actually this looks reasonable. All this is doing is determining if the questionnaires can be copied to the new proposal. Not if the questionnaire is complete or not.